lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -25,6 +26,11 @@ export class BrowserUtils {
         } else {
             window.location.assign(urlNavigate);
         }
+
+        // To block code from running after navigation, this should not throw if navigation succeeds
+        setTimeout(() => {
+            throw BrowserAuthError.createNavigationFailedError();
+        }, BrowserConstants.NAVIGATION_TIMEOUT_MS);
```

Comment:
Also, 5 seconds may not be enough time...

Reply:
Rather than a constant can we make it a config param?

Reply:
I think throwing here makes it clear what happened rather than silently failing (meaning we failed to navigate away from the page). Is there a scenario you can think of where we would not want to throw here? We could make the timeout something much bigger, and configurable, if the concern is the potential of taking longer to leave the page.

Reply:
Changed to a log message instead of an error. Making it configurable is a little messy and maybe not necessary if we set the timeout to something fairly large (I set it to 30 seconds now, but feel free to suggest something different)

Reply:
I still think making it configurable is the right way to go but if we want to leave it for a later PR that's fine.

Reply:
Was able to make the timeout configurable